### ISSUE TITLE
The parameter settings of the “source only” model and why the I^3 model is only trained for 70 epochs to achieve the optimal?

### ISSUE THREAD

**Issue by life97** [czzbb/I3Net#3]
Hello, thank you very much for providing the source code for learning. Does the experiment “source only” in the paper mean directly using VOC training and then using the target domain test set for testing? How are the parameters of the SSD training VOC dataset set? Is it also training for 70 epochs? What is the learning rate? (I used the VOC data set to train the SSD model, and the loss appeared NAN when training with the default settings) Finally, why is the model trained for 70 epochs? Thank you very much for your paper and source code, which inspired me a lot, and look forward to your reply.

**Comment by life97** [comment]
Thanks for providing such a good work for study, it is very inspiring to me.

**Comment by juncaoker0r0** [comment]
Hi @life97 ,were you able to solve this? I seemed to get loss == NAN too

**Comment by life97** [comment]
> Hi @life97 ,were you able to solve this? I seemed to get loss == NAN too

Sorry, it's been so long, I've forgotten how to fix this, and haven't taken notes. This model source code can indeed be trained, sorry for not being able to help you.
